Source/control.cpp

```diff
@@ -981,6 +981,11 @@ void DoAutoMap()
 	}
 }
 
+bool CanPanelsCoverView()

```

Comment:
nit: best place this in the header to let the compiler inline this easily

functions in the header must be marked `inline`